src/main.js

```diff
@@ -67,6 +68,10 @@ store.commit('loadLocalCredentials', SERVER_AUTHENTICATED);
 
 // Before loading a new route check if the user is authorised
 router.beforeEach((to, from, next) => {
+  if (to.path === '/develop/data') {
```

Comment:
I wonder if `from.path === '/schema/design'` would be more robust?

Reply:
fixed.